src/main/kotlin/com/cognifide/gradle/environment/docker/Docker.kt

```diff
@@ -14,7 +14,9 @@ class Docker(val environment: EnvironmentExtension) {
 
     val running: Boolean get() = stack.running && containers.running
 
-    val up: Boolean get() = composeFile.get().asFile.exists() && stack.running && containers.up
+    val up: Boolean get() = stack.running && containers.up
+
+    val upToDate get() = composeFile.get().asFile.exists()
```

Comment:
TODO introduce checksum / file content based comparison of already deployed docker-compose.yml file and with newly generated one. reload on any difference.

Reply:
done